packages/generator-widget/CHANGELOG.md

```diff
@@ -6,6 +6,8 @@ The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/),
 
 ## [Unreleased]
 
+### Changed
+
 - We've updated our templates to support the new version of Pluggable Widgets Tools containing new React and React Native versions. This is necessary due to breaking changes introduced in Studio Pro 9.24.0.
```

Comment:
🎄If you're going to touch this anyway, we can put three spaces after the `-` like everywhere else :P.

Reply:
Fixed ✨